### ISSUE TITLE
enable build pull request previews

### ISSUE THREAD

**Issue by raoufchebri** [scylladb/care-pet#77]
Following up #76 
Once this PR is merged, @tzach should follow [Enable Previews](https://github.com/scylladb/sphinx-scylladb-theme/blob/8316b705d909b695aa2765a5826a53be20e020e9/docs/source/deployment/previews.rst#enable-previews) guide to let AWS know you want to enable build pull request previews for this repository.

**Comment by tzach** [comment]
I follow the process, enabled everything and sent a new PR https://github.com/scylladb/care-pet/pull/78
